### PR TITLE
5 new plugins to include in the index

### PR DESCRIPTION
**Pull request by n0guest**
Hello.

This PR contains information about following plugins:

* [OSSHelp/drone-alertmanager](https://hub.docker.com/r/osshelp/drone-alertmanager)
* [OSSHelp/drone-git-mirror](https://hub.docker.com/r/osshelp/drone-git-mirror)
* [OSSHelp/drone-linter](https://hub.docker.com/r/osshelp/drone-linter)
* [OSSHelp/drone-molecule](https://hub.docker.com/r/osshelp/drone-molecule)
* [OSSHelp/drone-prometheus-alerts](https://hub.docker.com/r/osshelp/drone-prometheus-alerts)

Plus new logo for:

- Prometheus

Please, include these in the public catalog of plugins for Drone CI. If there is anything needed to be changed, please, let me know.

Thank you.

### CODE REVIEW

**Comment by n0guest** [comment]
@bradrydzewski or @tboerger, could someone, please, take a look at this PR? In short - we want to add few new plugins to the public index.

Thank you.

File: content/OSSHelp/drone-alertmanager/index.md
Comment (appleboy) [inline content/OSSHelp/drone-alertmanager/index.md:31]: remove the line?

**Review by appleboy — CHANGES_REQUESTED** [review]

File: content/OSSHelp/drone-alertmanager/index.md
Comment (n0guest) [reply content/OSSHelp/drone-alertmanager/index.md]: Done

**Review by appleboy — APPROVED** [review]
LGTM. need other maintainer approval.

**Comment by d1wilko** [comment]
Hi,

We have recently updated the drone plugin index and docs are no longer written in markdown - they are now a more consistent yaml format e.g. [docker](https://github.com/drone/drone-plugin-index/blob/master/plugins/docker/content.yaml)

Please have a look at the new readme if you would like to run the new plugin index [readme](https://github.com/drone/drone-plugin-index/blob/master/README.md)

If you would still like to add documentation to the plugins index can you change the format to
- An individual folder per plugin inside the `plugins` folder
- a `content.yaml` file that follows the format in the example above - this is for quick discovery, not full documentation
- an `original.md` file that can contain the markdown you have already written
- a logo image inside the `public/logos` folder

Thanks :)

**Pull request closed without merging**
